Doc: Pacemaker Remote: Update for AlmaLinux 9, and assorted fixes

nrwahl2 opened the pull request:
Closes [T492](https://projects.clusterlabs.org/T492).

I've got other minor fixes and improvements on the to-do list. This covers the meat of it.

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/intro.rst:
```diff
@@ -131,14 +131,14 @@ nodes.
 deployment we test in our developer environment to verify guest node scalability:
 
-* 16 cluster nodes running the full Corosync + Pacemaker stack
+* 32 cluster nodes running the full Corosync + Pacemaker stack
```
No idea what's actually in our developer scalability testing environment.

nrwahl2 replied:
done

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/baremetal-tutorial.rst:
```diff
@@ -79,32 +93,64 @@ Install the pacemaker_remote daemon on the remote node.
+
+    [root@pcmk-1 ~]# pcs host auth remote1 -u hacluster
+    Password: 
+    guest1: Authorized
```
whoops

nrwahl2 commented:
Replaced commits but **no** substantive changes. Added missing blank line in "Fix guest node integration" commit. Replaced two stray "guest" occurrences in the "Fix remote node integration" commit. Changed an IP address that doesn't come up anywhere else, in the 8->9 commit.

kgaillot commented on doc/sphinx/Pacemaker_Remote/baremetal-tutorial.rst:
```diff
@@ -72,7 +72,7 @@ node.
 
     # cat << END >> /etc/hosts
-    192.168.122.10  remote1
+    192.168.122.103  remote1
```
I'm not sure the IP address changes belong in this commit -- are they related to the OS switch?

BTW the rest of this commit makes sense as a single commit since everything is related and necessary for it to work

nrwahl2 replied:
Nah, I'll remove.

kgaillot reviewed (commented):
Nice update

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst:
```diff
@@ -98,14 +98,11 @@ vice versa.
-we also install the ``pacemaker`` package; it is not required, but
-it contains the dummy resource agent that we will use later
-for testing.
+Install the pacemaker_remote daemon on the guest node.
```
Will do. `pacemaker` should pull in `pacemaker-cli` regardless. `crm_attribute` is a better reason though.

At that point, if we install pacemaker, the switch from pacemaker Dummy to heartbeat Dummy isn't needed anymore. I'm inclined to leave it just to save a bit of time.

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst:
```diff
@@ -45,27 +45,35 @@ Create Guest
 Here's an example way to create a guest:
 
-* Download an .iso file from the `CentOS Mirrors List <http://isoredirect.centos.org/centos/8-stream/isos/x86_64/>`_ into a directory on your cluster node.
+* Download an .iso file from the |CFS_DISTRO| |CFS_VERSION| `mirrors list
```
Ah yep, sloppy mistake. I messed up `CFS_DISTRO_VER` once in the CfS doc too. Will fix that in this PR as well.

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst:
```diff
@@ -113,27 +126,120 @@ cluster. It is incredibly simple.
+    implicit resource will be named with the value of the ``VirtualDomain``
+    resource's ``remote-node`` meta attribute, which will be set by ``pcs`` to
+    the guest node's uname. Therefore, that value cannot be used as the name of
+    any other resource.
```
Done. In this case I was copy-pasting the existing text.

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst:
```diff
@@ -113,27 +126,120 @@ cluster. It is incredibly simple.
+    the guest node's uname. Therefore, that value cannot be used as the name of
+    any other resource.
+
+Now we can confirm that the ``VirtualDomain`` resource is running on ``pcmk-1``.
```
Done

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst:
```diff
@@ -436,118 +556,6 @@ and a cluster node exists. This is particularly important for any promotable
-
-  .. code-block:: none
-
-    # virsh dumpxml guest1 > /etc/pacemaker/guest1.xml
```
They don't, although they could be updated to do so.

The `virt-install` command is what creates the XML IIRC. It has a `--print-xml` option that could be redirected to another file. Not sure off the top of my head how that would impact the disk path... it says `--print-xml` **does** still do storage creation by default.

Can add to the to-do list if you want.

kgaillot replied:
No, it's not important

nrwahl2 commented on doc/sphinx/Pacemaker_Remote/kvm-tutorial.rst:
```diff
@@ -17,20 +17,6 @@ Configure Cluster Nodes
-    # cat << END >> /etc/hosts
-    192.168.122.10    guest1
-    END
-
```
Agreed, done.

nrwahl2 commented:
I cannot get rid of these damn commits...

nrwahl2 commented:
Updated to address review.
* Reverted 32 -> 16 for "our developer environment" and changed description to reflect that there is no "our developer environment."
* Re-added the command to install `pacemaker` on the guest, but referred to `crm_attribute` in the explanation.
* `s/uname/node name/` re: guest
* added `--wait` to `cib-push` command. May want to do that later for CfS too.
* Removed IP address change altogether. I feel like a different subnet is more reasonable for a KVM guest, but that's something users can determine for themselves. No real reason to change it in the doc.
* Broke "Fix guest node integration" mega-commit into several self-contained commits.
